unlock-app/src/components/creator/CreatorLockForm.js

```diff
@@ -44,6 +45,11 @@ class CreatorLockForm extends React.Component {
     if (this.props.hideAction) this.props.hideAction()
   }
 
+  handleCancel(e) {
+    e.stopPropagation()
```

Comment:
Pre-emptive: we need this line to prevent the submit method from _also_ being called.